styles/mixins/triangle.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = function( config ) {
```

Comment:
are we able to make a triangle like this with this mixin?

![screenshot 2016-09-05 at 8 44 49 pm](https://cloud.githubusercontent.com/assets/27936/18261168/a2ac5dd0-73a9-11e6-901a-4b7c0344b2fa.png)